src/exceptions/global-exceptions.filter.ts

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable max-lines-per-function */
+import {
+  ArgumentsHost,
+  Catch,
+  ExceptionFilter,
+  HttpException,
+  HttpStatus,
+  Logger,
+} from '@nestjs/common';
+import { Request, Response } from 'express';
+import { EntityNotFoundError, QueryFailedError } from 'typeorm';
+
+import { ClientApiResponse } from '../types/client-api/client-api.response';
+import { ErrorData } from '../types/error/error-data';
+import { MysqlErrorCodes } from '../types/error/mysql-errors';
+import { ErrorMessage } from '../types/messages/error-message/error-message';
+import { MySqlErrorMessage } from '../types/messages/mysql-message/mysql-error.message';
+
+@Catch()
+export class GlobalExceptionFilter implements ExceptionFilter {
+  catch(exception: unknown, host: ArgumentsHost): void {
+    const ctx = host.switchToHttp();
+    const response = ctx.getResponse<Response<ClientApiResponse<null>>>();
+    const request = ctx.getRequest<Request>();
+
+    const { status: statusCode, ...errorData } = this.tryToGetErrorData(exception);
+    Logger.log({
```

Comment:
Tu podobnie. Trzeba by było poczytać (chyba że się wdrożyłeś w to) I ustalić jaki typ logowania do jakich danych